style.css

```diff
@@ -0,0 +1,584 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-family: 'lato', sans-serif;
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  width: 100vw;
+}
+
+/* Header and Section 1 */
+
+.header-section1-container {
+  background-image: url('images/home-bg-mobile.png');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  padding: 20px 5vw;
+}
+
+/* Header */
+
+.header .hamburger {
+  display: inline-block;
+  cursor: pointer;
+}
+
+.header .hamburger span {
+  display: block;
+  height: 5px;
+  width: 25px;
+  margin-bottom: 5px;
+  background: #333;
+}
+
+.header .mobile-nav {
+  z-index: 100;
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: none;
+  width: 100vw;
+  height: 100vh;
+  justify-content: space-between;
+  background: #e4e4e4;
+  font-weight: 900;
+  font-size: 40px;
+  line-height: 60px;
+}
+
+.header .mobile-nav .a {
+  position: relative;
+  left: 20px;
+  top: 50px;
+  color: rgb(111, 110, 110);
+}
+
+.header .mobile-nav .closeBtn {
+  color: grey;
+  position: relative;
+  right: 20px;
+  cursor: pointer;
+}
+
+.header .mobile-nav .a:hover,
+.header .mobile-nav .closeBtn:hover {
+  color: #ec5242;
+}
+
+.header .mobile-nav .li-cc {
+  margin-top: 30px;
+}
+
+.header .mobile-nav .a-cc {
+  border: solid 4px red;
+  padding: 10px;
+  color: #ec5242;
+  white-space: nowrap;
+}
+
+.header .desktop-nav {
+  display: none;
+}
+
+/* Section 1 */
+
+.section-1 {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] I'd recommend you use selector classes multiple times rather than repeating rules, as this keeps your style code DRY and easier to maintain. I noticed you're using `display: flex;` multiple times in different selectors so instead of doing that you can create a class like the one below and use that class name on any HTML element that uses that styling.
```
.flex-col {
  display: flex;
  flex-direction: column;
}
.flex-row {
  display: flex;
  flex-direction: row;
}
```